docs/resources/tutorials/core/qos/mutable/profiles1.xml

```diff
@@ -0,0 +1,22 @@
+    <?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I think this files can have a more verbose name; something like `large_ownership_strength` perhaps?

Reply:
Done